python/whylogs/core/metrics/metrics.py

```diff
@@ -334,7 +336,10 @@ def min(self) -> float:
 
     @classmethod
     def zero(cls, config: MetricConfig) -> "DistributionMetric":
-        sk = ds.kll_doubles_sketch(k=config.kll_k)
+        if config.customize_kll:
+            sk = ds.kll_doubles_sketch(k=config.kll_k_custom)
+        else:
+            sk = ds.kll_doubles_sketch(k=config.kll_k)
```

Comment:
Keeping the names the same for the suggestion but update those if you decide to based on comment above.
```suggestion
        configured_kll_k = config.kll_k_custom if config.customize_kll else config.kll_k
        sk = ds.kll_doubles_sketch(k=configured_kll_k )
```

Reply:
added changes in the last commit